packages/bpk-component-breadcrumb/src/BpkBreadcrumbItem.js

```diff
@@ -0,0 +1,108 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import PropTypes from 'prop-types';
+import React, { type Node } from 'react';
+import BpkLink from 'bpk-component-link';
+import BpkText from 'bpk-component-text';
+import { cssModules } from 'bpk-react-utils';
+import { withRtlSupport } from 'bpk-component-icon';
+import Chevron from 'bpk-component-icon/sm/arrow-right';
+
+import STYLES from './BpkBreadcrumbItem.scss';
+
+const getClassName = cssModules(STYLES);
+
+export type Props = {
+  children: Node,
+  active: boolean,
+  href: ?string,
+  className: ?string,
+  accessibilityLabel: string,
```

Comment:
Name aside, there is a better way to declare the flow type for this. Check out the [badge](https://github.com/Skyscanner/backpack/blob/master/packages/bpk-component-badge/src/BpkBadge.js#L48) as an example. 

We declare the "enumeration" in a plain ole JS obj:
```js
export const ARIA_CURRENT = {
  page: 'page',
  step: 'step',
  location: 'location',
  date: 'date',
  time: 'time',
  true: true,
  false: false,
};
```

Then the flow type can be expressed like so:
```js
current: $Keys<typeof ARIA_CURRENT>,
```

The prop type can e expressed like so:
```js
current: PropTypes.oneOf(Object.keys(ARIA_CURRENT)),
```